test/acceptance/behave/README.md

```diff
@@ -62,15 +66,19 @@ Note: We recommend the use of virtualenv, because is an isolated working copy of
 ### Tests execution:
 
 - Change to the test/acceptance/behave folder of the project if not already on it.
-- You may need to set `export GIT_SSL_NO_VERIFY=true` environment variable in your machine
-- You should add "--upgrade" (if you have a previous installed version of this library)
-- We recommend to create `settings` folder in  behave root directory if it does not exists and store all configurations to referenced by `properties.json` files.
+- We recommend to create `settings` folder in  behave root directory if it does not exists and store all configurations to `properties.json` and `configuration.json` files.
   The settings folder path could be changed in the `configuration.json`.
   This file initially will overwrite properties.json in each feature.
 - modify in `configuration.json`:
-       * JENKINS: determine whether you are in jenkins or not.
        * PATH_TO_SETTING_FOLDER: folder where are the different configurations
-- `properties.json` will be update automatically from settings folder (see configuration.json)
+       * UPDATE_PROPERTIES_JSON: determine whether you are create/update `properties.json` automatically or manually (used to jenkins).
+           - true: read external file (the file is mandatory) in `PATH_TO_SETTING_FOLDER` (with ssh commands) and execute these automatically (used to create `properties.json`) 
+           - false: it does nothing and the creation of `properties.json` will be by user in jenkins console.
+       * CB_RUNNING_MODE: is used to determine how is compiled ContextBroker.
+           ContextBroker will be compiled and installed previously by user, `http://fiware-orion.readthedocs.org/en/develop/admin/build_source/index.html`.        
+           - RPM: CB is installed as RPM, so service tooling will be used to start and stop
+           - CLI: plain CB command line interface will be used to start contextBroker (must be compiled from source in DEBUG mode)               
+- `properties.json` (MANDATORY) will be create/update automatically from settings folder (see `configuration.json`) or manually.
 - Run behave (see available params with the -h option).
 ```
     Some examples:
```

Comment:
I would suggest the way in which examples are shown:

Instead of

```
behave .../example.feature
```

maybe is clearer:

```
behave components/path/to/example.feature
```